examples/test_api.c

```diff
@@ -747,24 +764,25 @@ int main (int argc, char* argv[])
   MPI_Comm_rank(MPI_COMM_WORLD, &rank);
   MPI_Comm_size(MPI_COMM_WORLD, &ranks);
 
-  static const char *opt_string = "s:t:z:w:p:f:o:a:c:h";
+  static const char *opt_string = "s:t:z:w:p:f:o:a:c:h:k:";
   static struct option long_options[] = {
     {"size",    required_argument, NULL, 's'},
     {"times",   required_argument, NULL, 't'},
-    {"seconds", required_argument, NULL, 'z'},
+      {"seconds", required_argument, NULL, 'z'},
```

Comment:
I'm not sure where the additional spaces are coming from, please remove them

Reply:
FIXED